bbn-test-5/babylon-validators/README.md

```diff
@@ -0,0 +1,358 @@
+# Babylon Validator Setup
+
+## Table of Contents
+
+1. [Prerequisites](#1-prerequisites)
+2. [System Requirements](#2-system-requirements)
+3. [Key Management](#3-key-management) 
+   1. [Key for CometBFT consensus](#31-key-for-cometbft-consensus)
+      1. [Get Funds](#311-get-funds)
+   2. [Key for BLS Voting](#32-key-for-bls-voting)
+      1. [What is BLS Voting](#321-what-is-bls-voting)
+      2. [Create BLS Key](#322-create-bls-key)
+4.  [CometBFT Validator Configuration](#4-cometbft-validator-configuration)
+5. [Creating a Validator](#5-creating-a-validator)
+   1. [Verifying Validator Setup](#51-verifying-validator-setup)
+   2. [Understanding Validator Status](#52-understanding-validator-status)
+   3. [Managing Your Validator](#53-managing-your-validator)
+6. [Advanced Security Architecture](#6-advanced-security-architecture)
+7. [Conclusion](#7-conclusion)
+<!-- 7. [Enhanced Monitoring](#7-enhanced-monitoring)
+   1. [Prometheus Configuration](#71-prometheus-configuration)
+   2. [Basic Health Checks](#72-basic-health-checks) -->
+
+
+## 1. Prerequisites
+
+Before setting up a validator, you'll need a fully synced Babylon node. For 
+node setup instructions, see our [Node Setup Guide](../babylon-node/README.md).
+
+## 2. System Requirements
+
+Recommended specifications for running a Babylon validator node:
+- CPU: Quad Core AMD/Intel (amd64)
+- RAM: 32GB
+- Storage: 2TB NVMe
+- Network: 100MBps bidirectional
+- Encrypted storage for keys and sensitive data
+- Regular system backups (hourly, daily, weekly)
+
+These are reference specifications for a production validator. 
+Requirements may vary based on network activity and your operational needs.
+
+## 3. Key Management
+### 3.1 Key for CometBFT consensus
```

Comment:
I think we are missing something here. The key used for CometBFT consensus is the one that is stored on `<home>/config/priv_validator_key.json`. Did a minor test and it seems that this file is created when the user does `babylond init` to iinitialize their home directory.

The above means that the key for the CometBFT is already on `priv_validator_key.json` when the user reaches this section. Instead, the instructions that we are giving here are about creating a key in the application layer (i.e. a Babylon account) to be associated with the CometBFT key to be used for Consensus.

Therefore I recommend we structure section 3 as follows:

- 3.1.: Key for CometBFT consensus: Contains small note about the key being in the `priv_validator_key.json` file when initializing.
- 3.2.: Babylon validator account keyring: The below section as is with some additional note on this being the application layer account associated with the CometBFT key.
- 3.3.: BLS key: same as it is